planning/behavior_path_planner/src/scene_module/start_planner/start_planner_module.cpp

```diff
@@ -752,6 +731,7 @@ void StartPlannerModule::updateStatusAfterBackwardDriving()
 {
   status_.driving_forward = true;
   status_.backward_driving_complete = true;
+  incrementPathIndex();
```

Comment:
`status_.pull_out_path.partial_paths` does not contain the backward path, so incitement is maybe not needed
did you check with `geometric_pull_out` ? this may skip the first arc path

Reply:
@kosuke55 
Thank you for checking!
I deleted this incrementPathIndex(); and confirmed in the situation.

[Screencast from 2023年11月17日 13時43分19秒.webm](https://github.com/autowarefoundation/autoware.universe/assets/32741405/dae28da6-b727-4d44-8bf9-f263a1a27ec3)